api/migrations/versions/38dde1052c5e_.py

```diff
@@ -0,0 +1,56 @@
+"""empty message
+
+Revision ID: 38dde1052c5e
+Revises: 351c26992f23
+Create Date: 2019-01-23 19:56:29.948294
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '38dde1052c5e'
+down_revision = '351c26992f23'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import table
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    event = op.create_table('Event',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('Name', sa.String(length=255)),
```

Comment:
can we still with the convention of small letters for our column names?

Reply:
Done :)